tendrl/ceph_integration/objects/definition/ceph.yaml

```diff
@@ -210,9 +222,21 @@ namespace.ceph:
           enabled: true
           inputs:
             mandatory:
-              - Rbd.pool_id
               - Rbd.name
               - Rbd.size
+            optional:
```

Comment:
@r0h4n The API can handle the mandatory and optional parameters, no code change is needed to handled additional attributes too.

The mandatory and optional attributes are available via the /flows API which can be used to find the mandatory and optional parameters to build the UI.